sdk/eventhub/azure-eventhub/azure/eventhub/_pyamqp/aio/_client_async.py

```diff
@@ -450,101 +470,17 @@ async def _send_message_impl_async(self, message, **kwargs):
 
     async def send_message_async(self, message, **kwargs):
         """
-        :param ~uamqp.message.Message message:
+        :param ~pyamqp.message.Message message:
         :param int timeout: timeout in seconds
         """
         await self._do_retryable_operation_async(self._send_message_impl_async, message=message, **kwargs)
 
 
+SendClientAsync.__doc__ = \
+    f"""{SendClientSync.__doc__[0:41]} asynchronously{SendClientSync.__doc__[41::]}"""
```

Comment:
Will we need to keep updating these index numbers each time we modify the baseclass docstring?